app/src/common/shared/org/mozilla/vrbrowser/telemetry/TelemetryWrapper.java

```diff
@@ -132,6 +134,7 @@
         private static final String THREE_OPEN_WINDOWS_TIME_PCT = "three_windows_open_time_pct";
         private static final String WINDOWS_OPEN_W_MEAN = "window_open_w_mean";
         private static final String WINDOWS_PRIVATE_OPEN_W_MEAN = "window_open_private_w_mean";
+        private static final String TELEMETRY_STATUS = "telemetry_status";
```

Comment:
Please try to avoid the string length be longer than 15. The key will be cut in our database.